apps/notifications/src/panel/templates/button-answer-prompt.jsx

```diff
@@ -9,7 +9,10 @@ import ActionButton from './action-button';
 // eslint-disable-next-line no-shadow
 const AnswerPromptButton = ( { answerPrompt, note, translate } ) => {
 	const { site: siteId } = note?.meta?.ids ?? {};
-	const newPostLink = getNewPostLink( note );
+	const isLocalhost = document.location.hostname === 'calypso.localhost';
```

Comment:
I think we should be use protocol + host like they do here https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/composite-checkout/hooks/use-strip-products-from-url.ts#L26

That way it will work with calypso.live environments etc and not always take us to wordpress.com

```suggestion
	const newPostLink =  
	    document.location.protocol + 
	    '//' + 
	    document.location.host + 
	    getNewPostLink( note );
```

Reply:
Ah nice, that is better. Thanks.